### ISSUE TITLE
uses the same icon as gnome-calculator

### ISSUE THREAD

**Issue by Nine-H** [elementary/calculator#22]
I think a branded icon would be nice, either that or a full scientific and programmer mode.

**Comment by cassidyjames** [comment]
The system calculator uses the system's FreeDesktop.org calculator icon. I don't believe this is a bug.

**Comment by danrabbit** [comment]
Yeah closing as "won't fix"

**Comment by Nine-H** [comment]
sorry, filed this at 1am from the other side of the planet: you guys are graphic designers, let me rephrase this as a visual communications problem: without clicking, which calculator is your calculator?

![screenshot from 2017-06-30 13 34 29](https://user-images.githubusercontent.com/20123543/27719956-702b4b16-5d99-11e7-90f5-1243c8910dfe.png)

**Comment by danrabbit** [comment]
The problem is that from our perspective, GNOME Calculator is the third party app. That app should be using its own branded icon, not our app.

**Comment by Nine-H** [comment]
That's a fair point, and I'm aware that for projects like this content can be more difficult than code. This issue irks me and it's more annoying when using search. Politically it's annoying too, because upstream is even less likely to address it. There's plenty of workarounds from a user perspective but I'm not sure that having users edit /usr/share/applications/*.desktop or remove default apps is a clean solution. A branded icon wouldn't have to be a radical change: different colors that reflect the pantheon calculator window design would be more than enough.
